.github/workflows/build-on-windows.yml

```diff
@@ -170,6 +170,27 @@ jobs:
           name: test-results
           path: test_results_*.xml
 
+      - name: 🔽 Install 'OpenCppCoverage' and 'codecov.io Report Uploader'
+        run: |
+          curl -L -O https://github.com/OpenCppCoverage/OpenCppCoverage/releases/download/release-0.9.9.0/OpenCppCoverageSetup-x64-0.9.9.0.exe
+          OpenCppCoverageSetup-x64-0.9.9.0.exe /VERYSILENT /DIR=.\bin\coverage
+          curl -L https://uploader.codecov.io/latest/windows/codecov.exe --output bin\coverage\codecov.exe
+
+      - name: 📑 Generate CodeCoverage Report
+        run: |
+          .\bin\coverage\OpenCppCoverage.exe --sources=src        ^
+          --excluded_sources=src\*_tests.cpp                      ^
+          --excluded_sources=build\_deps                          ^
+          --export_type=cobertura:ttauri_coverage.xml             ^
+          --working_dir=.\\build\ttauri-%VERSION%\tests           ^
+          -- ".\\build\ttauri-%VERSION%\tests\\ttauri_tests.exe"
+
+      - name: 📦 🚀 Upload CodeCoverage Report to codecov.io
```

Comment:
Since this item needs a secret, I think it can only run from 'main'. Maybe lets add a "if" statement on these items?

Reply:
I was unsure, if the `CODECOV_TOKEN` is needed. But uploading works fine without it. 
So i just removed the line.

Reply:
I think I am happy with Debug and just main-branch.
I believe i only want to use this to see overall progress of the project, not on a pull request.